Makefile

```diff
@@ -25,63 +25,58 @@ LDFLAGS += -X "github.com/rancher/elemental-operator/pkg/version.Commit=${GIT_CO
 LDFLAGS += -X "github.com/rancher/elemental-operator/pkg/version.CommitDate=${COMMITDATE}"
 
 # Directories
-BIN_DIR := bin
-TOOLS_DIR := hack/tools
-TOOLS_BIN_DIR := $(abspath $(TOOLS_DIR)/$(BIN_DIR))
+BUILD_DIR := build
```

Comment:
uh, I would leave the build dir alone, its where we spit our stuff like charts and artifacts, it may collide with some of the release jobs...

Reply:
sure, I changed the path for tools